client/src/components/MusicPlayerPanel.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useRef, useState } from 'react';
+import { StyledButton } from './shared/StyledButton';
+import { makeStyles } from '@material-ui/core/styles';
+import { TextField, Button } from '@material-ui/core';
+import { IMusicPlayer } from '../types';
+
+const useStyles = makeStyles({
+	container: {
+		display: 'flex',
+		alignItems: 'center',
+		justifyContent: 'center',
+		padding: 10,
+		background: 'var(--background)',
+		width: '100%',
+		'& > *': {
+			marginRight: 10
+		}
+	},
+	input: {
+		borderRadius: 20,
+		background: 'white'
+	}
+});
+
+interface IMusicPlayerPanelProps {
+	changePlaylist: (message: string) => void;
+	musicPlayer: IMusicPlayer;
+}
+
+export const MusicPlayerPanel = ({
+	changePlaylist,
+	musicPlayer
+}: IMusicPlayerPanelProps) => {
+	const classes = useStyles();
+
+	const [urlValue, setUrlValue] = useState('');
+	const textfieldRef = useRef<HTMLDivElement>(null);
+
+	const onChangeUrl = (event: React.ChangeEvent<HTMLInputElement>) => {
+		setUrlValue(event.target.value);
+	};
+
+	const onKeyPressEnter = (event: React.KeyboardEvent<HTMLInputElement>) => {
+		if (event.key === 'Enter') {
+			addMusic();
+			setUrlValue("");
+		}
+	};
+
+	const onFocus = () => {
+		if (window.innerWidth < 500 && textfieldRef.current) {
```

Comment:
Avoid using magic numbers like 500. For code clarity you should declare the number as a variable like 
`const SMALL_SCREEN_WIDTH = 500`